### ISSUE TITLE
Accessibility - Left menu is after main content

### ISSUE THREAD

**Issue by jeanbisson** [wet-boew/cdts-sgdc#873]
There is an accessibility issue with left menu (GoCWebTemplate.Application.LeftMenu_Layout.cshtml)
v4_0_32 and v4_0_45

"Issue: In the HTML code, the order of content is not in the same order as visually presented, which prevents a meaningful sequence experience for some users. In the code, the left side menu comes after the main content, which means that people relying on screen readers to get by or people unable to use a mouse can't get to this menu until they have tabbed through the whole page at the end.

How to fix: Make sure that in the HTML source code the left side menu comes before the page's main content."
[https://esdc.prv/cgi-bin/scdtic-ictdcs/page-error-view.aspx?id=143&wid=0](https://esdc.prv/cgi-bin/scdtic-ictdcs/page-error-view.aspx?id=143&wid=0)

Also when the screen size is small, the left menu shows up visually at the bottom after the main content (which might be ok, it wasn't mentioned)

**Comment by ahmad-shahid** [comment]
Hello, thank you for creating this issue. The left menu layout will be fixed in the .net templates repo. 

I created the issue: https://github.com/wet-boew/cdts-DotNetTemplates/issues/42
